tasks/Bartender.tex

```diff
@@ -0,0 +1,29 @@
+\section{Bartender [Party Host]}
+The home owner will teach the robot to make a drink which guests at the party really like!
+
+\subsection{Focus}
+Learning from demonstration.
+
+\subsection{Main Goal}
+The home owner will teach the robot using demonstrations how to make a drink. The robot is then to repeat the task. A designated team member will show the home owner how to perform the demonstration. These drinks need to be \textit{good}, meaning that the robot must pour each mixer in for the right amount of time to get the right amount of mixer in.
```

Comment:
I find this overly complex, not to say impossible, for the following reasons:
1. We haven't addressed blenders or any other type of domestic appliance before.
2. We haven't handled liquids as part of regular tests before, let alone pouring them.
3. We haven't addressed properly pouring solids.
4. Measuring amounts of is still out of the chart.
5. Having an HSR handling liquids voids the warranty.

Further, the main goal is too complex. Short version required. Suggested:

> The ~home owner~ operator teaches the robot how to make a drink by demonstration. Once the operator has finished the demonstration, the robot has to repeat the same procedure it observed.

The team has no saying in this. The robot must clearly state how the operator must trigger the learning behavior. It must be natural.

Finally, I lack dialogue components people often used when instructed. After learning how to make a Mojito I had several questions so I asked.